Feed taxii2 exception handling (#39347)
demisto/content: Packs/FeedTAXII/ReleaseNotes/1_2_34.md

#### Integrations

##### TAXII 2 Feed

> - Fixed an issue where an error in one collection would halt the fetch process. The fetch now continues with the remaining collections even if one fails.

Reviewer: ```suggestion
- Fixed an issue where an error occurring in one collection previously halted the overall fetch process.
```
